### PR TITLE
improve darwin

### PR DESCRIPTION
**Pull request by mskonovalov**
This fixes compilation error

Plus it enriches Darwin info from sw_vers tool
Version numbers are from wikipedia page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,12 +36,14 @@ OSinfo is a cross-platform OS Version collection tool. It is designed to unify m
...
    import (
-	   "github.com/JustinTimperio/osinfo"
+   	    "fmt"
+   	    
+	    "github.com/JustinTimperio/osinfo"
    )
 
    func main() {
 		release := osinfo.GetVersion()
-		release.PrintInfo()
```
Comment (JustinTimperio) [inline README.md:44]: Why change this?

File: osinfo.go [hunk 2]
```diff
@@ -14,26 +14,31 @@ type Release struct {
...
-	Linux   linuxRelease
-	BSD     bsdRelease
+	Windows WindowsRelease
+	Linux   LinuxRelease
+	BSD     BsdRelease
+	MacOs   MacOsRelease
 }
 
-type windowsRelease struct {
+type WindowsRelease struct {
```
Comment (JustinTimperio) [inline osinfo.go:23]: This struct should not be exported. This is accessible with `release.Windows.<somekey>`

File: osinfo.go [hunk 3]
```diff
@@ -14,26 +14,31 @@ type Release struct {
...
 	Build string
 }
 
-type linuxRelease struct {
+type LinuxRelease struct {
```
Comment (JustinTimperio) [inline osinfo.go:27]: This struct should not be exported. This is accessible with `release.Linux.<somekey>`

File: osinfo.go [hunk 4]
```diff
@@ -14,26 +14,31 @@ type Release struct {
...
 	Distro     string
 	Kernel     string
 	PkgManager string
 }
 
-type bsdRelease struct {
+type BsdRelease struct {
```
Comment (JustinTimperio) [inline osinfo.go:33]: This struct should not be exported. This is accessible with `release.BSD.<somekey>`

**Comment by JustinTimperio** [comment]
Hi @mskonovalov thanks for the support with darwin! I left you some comments on your PR

**Comment by mskonovalov** [comment]
Ooops, didn't have time to address the comments :)

**Comment by JustinTimperio** [comment]
@mskonovalov No worries! i just went ahead and merged it and fixed my only small issues. I also added some docs and moved to a stable release for those who want to use it. Thanks for testing Darwin!

**Pull request merged**